packages/phpstan-rules/tests/Rules/RequireStringRegexMatchKeyRule/Fixture/NumericDimDirectNext.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\PHPStanRules\Tests\Rules\RequireQuoteStringValueSprintfRule\Fixture;
+
+use Nette\Utils\Strings;
+
+class NumericDimDirectNext
+{
+    private const REGEX = '#(a content)#';
+
+    public function run()
+    {
+        $matches = Strings::match('a content', self::REGEX);
+        echo 'a statement before';
+        echo $matches[1];
+    }
```

Comment:
There should be also another test fixture covering other name + multiple values:

```php
$result = Strings::match('a content', self::REGEX);
        echo 'a statement before';
        if ($result[1] === '...') {

        } else {
      		return $result[4];
        }
}
```

Reply:
implemented.